pkg/core/docs/src/core/docs.md

```
@@ -0,0 +1,38 @@
+Renders MkDocs content inside your app. Opens specific sections in new windows for contextual help.
+
+## Setup
+```go
+package demo
+import (
+	"github.com/Snider/Core"
+	"github.com/Snider/Core/docs"
+)
+
+core.New(
+	core.WithService(docs.Register), 
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)